bin/generate

```diff
@@ -5,11 +5,11 @@ import yaml
 
 
 def section_header(text, level=1):
-    return '{0} {1}'.format('#' * level, text)
+    return "{0} {1}".format("#" * level, text)
```

Comment:
this file was formatted by `black` (not clear to me why it wasn't before)

Reply:
Different version of black? May be good to update black on CI and locally to latest :)

Reply:
That's probably it. The version is now fixed to the latest one: 
https://github.com/dokku/ansible-dokku/pull/86/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9R22